linux_os/guide/system/accounts/accounts-restrictions/root_logins/no_direct_root_logins/ignition/shared.yml

```diff
@@ -0,0 +1,18 @@
+# platform = multi_platform_all
+apiVersion: machineconfiguration.openshift.io/v1
+kind: Ignition
```

Comment:
The Kind should be "MachineConfig", not "Ignition". 
Sorry, I know this is confusing. Ignition is the yaml-based language that is used under spec.config. But together with the other things, the spec makes up a "MachineConfig" Kubernetes type.

Reply:
Great work overall by the way. I'm testing the result with the operator now.

Reply:
It was actually a find-replace error. I will fix that.

Reply:
@jhrozek while OCP may aggregate these snippets into MachineConfig's, would it be reasonable to expect use cases where other Kubernetes distributions may take the Ignition scripts for other purposes?

Or in other words -- should these remain to be called Ignition scripts, and let downstreams such as OpenShift convert to product-specific terms like Machine Configs? Trying to avoid having remediation snippets for OCP vs Docker vs Pivotal vs Rancher if possible, by letting Ignition be the commonality between them.

Reply:
updated: chatted with @jhrozek and @JAORMX in person. Lose agreement that keeping ignition namespace might make sense as it will simplify code in the compliance operator.

@jhrozek double checking compliance-operator code and reporting back.

Reply:
Right, if we were to keep this as ignition, we would then want to keep only from line 10 to 18. And remove the headers. Those should be handled by the compliance-operator as an implementation detail.